### PR TITLE
replacing starts_with\? with start_with\?

### PR DESCRIPTION
**Pull request by darena-mdsol**

### CODE REVIEW

**Comment by jcarres-mdsol** [comment]
LGTM, I'll wait for @clabrunda to take alook

**Comment by jcarres-mdsol** [comment]
Wait, I spoke too soon, you should also add a line to the changelog and update the minor version so we can release a new version to rubygems.

**Comment by darena-mdsol** [comment]
@jcarres-mdsol updated the minor version (1.2.1) and add a changelog entry. I'm not sure the best way to express the change, so I can change it if it is not good enough.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 1.2.1
+* Replaces `starts_with?` with `start_with?` in DiceBag::AvailableTemplates.checker_within_given_gems?
```
Comment (clabrunda) [inline CHANGELOG.md]: This is what you changed but not why.
Maybe:
```
* Replaces `starts_with?` with `start_with?` to remove Rails dependency.
```

**Comment by jcarres-mdsol** [comment]
Done! https://rubygems.org/gems/dice_bag/versions/1.2.1

**Pull request merged**